### PR TITLE
Updated games list (manually for now, will try to automate soon)

### PR DESCRIPTION
**Pull request by arcataroger**
Added a few new games from Sept 28 GFN Thursday:

https://blogs.nvidia.com/blog/2023/09/28/geforce-now-thursday-sep-28/

I saw your other changes and notes, and will reply ASAP! (Probably next week, a bit sick this week)

Thanks!

### CODE REVIEW

File: getSteamIds.js [hunk 1]
```diff
@@ -77,7 +77,9 @@ const getSteamIds = async () => {
     return [...steamIdsOfGamesOnGeForceNow];
 };
 
+/*
 getSteamIds().then((ids) => {
-    //console.log(JSON.stringify(ids))
-    //console.log(`Total games listed: ${ids.length}`)
+    console.log(JSON.stringify(ids))
```
Comment (mefaba) [inline getSteamIds.js:82]: left over logs 😅

**Review by mefaba — APPROVED** [review]

File: getSteamIds.js (hunk 1 above)
Comment (arcataroger) [reply getSteamIds.js:82]: Fixed, thanks :)

**Comment by arcataroger** [comment]
Sorry, I've still been sick, will try to automate the update process soon!

**Comment by mefaba** [comment]
> Sorry, I've still been sick, will try to automate the update process soon!

Thats exciting !

**Pull request merged**